rllib/agents/ppo/ppo.py

```diff
@@ -53,7 +53,7 @@ class PPOConfig(TrainerConfig):
     """Defines a PPOTrainer configuration class from which a PPOTrainer can be built.
 
     Example:
-        >>> config = PPOConfig(kl_coeff=0.3).training(gamma=0.9, lr=0.01)\
+        >>> config = PPOConfig().training(gamma=0.9, lr=0.01)\
```

Comment:
yep, I caught that one in my PR as well (also still needs the import). I wonder why `doctest` doesn't complain about this in CI, tbh...